src/static/css/main.less

```diff
@@ -26,6 +24,12 @@
 // Icon font path
 @cf-icon-path:                  '../fonts';
```

Comment:
You could get rid of the extra space in here while you're in there if you want. It doesn't line up with anything.